dsl/properties/scripts/ImportMicroservices.groovy

```diff
@@ -0,0 +1,440 @@
+/**	Docker Compose File structure:
+ *
+ *	version:
+ *	'2'
+ *	services:
+ *   web:
+ *	    image: nginx:13.1
+ *	    ports:
+ *	     - "5000:5000"
+ *	    volumes:
+ *	- .:/code
+ *	redis:
+ *	image: redis
+ */
+
+public class ImportMicroservices extends EFClient {
+	def dockerClient
+    def pluginConfig
+    def clusterEndpoint
+	def composeConfig
+    def importedSummary = [:]
+	
+	static final String CREATED_DESCRIPTION = "Created by Container ImportMicroservices"
+	
+	def ImportMicroservices(def dockerClient, def pluginConfig, def composeConfig) {
+        dockerClient = dockerClient
+        pluginConfig = params.pluginConfig
+        accessToken = kubeClient.retrieveAccessToken(pluginConfig)
+		this.composeConfig = composeConfig
+        clusterEndpoint = pluginConfig.clusterEndpoint
+    }
+	
+	def buildServicesDefinitions(def projectName, def applicationName) {
+		def efServices = []
+		composeConfig.services.each { name, serviceConfig ->
+			def efService = buildServiceDefinition(name, projectName, applicationName, serviceConfig)
+			efServices.push(efService)
+        }
+		efServices
+	}
+	
+	def buildServiceDefinition(def name, def projectName, def applicationName, def serviceConfig) {
+        def efServiceName = name
+        def efService = [
+            service: [
+                serviceName: efServiceName
+            ],
+            serviceMapping: [:]
+        ]
+
+        // Service Fields
+		efService.service.defaultCapacity = serviceConfig.deploy?.replicas
+		efService.service.minCapacity = serviceConfig.updateConfig?.parallelism
+		
+		String[] imageInfo = serviceConfig.image?.split(':')
```

Comment:
Please look at imageParse method in EC-Kubernetes, there is a bit more complex logic in this. There is optional regsitry url and image can come without repository name.

Reply:
Done